### PR TITLE
Allow a "no-image" mode for mosviz

### PR DESCRIPTION
**Pull request by ibusko**
The Mosviz example notebook exercises this mode.

### CODE REVIEW

**Review by javerbukh — APPROVED** [review]
Looks good! I imagine the next step is to remove the image viewer but I understand that will be a significant amount of work and should be tracked separately.

**Comment by havok2063** [comment]
Since this was something I've requested, I've also tested this.  It minimally works.  I think the `images` keyword argument should be made optional, possibly with just `images=None`.   I'm not sure how easy it is to disable/enable viewers, but one could imagine a workflow where the image viewer is enabled/disabled when the `images` keyword parameter is specified or not.

**Review by rosteen — CHANGES_REQUESTED** [review]
Works for me, but I think I agree with @havok2063 that if the images are optional, the argument in `load_data` should be `images=None` so that it defaults to None instead of needing explicit input of `None`.

And as @javerbukh said, potentially removing the image viewer in this case is on our radar but was decided to be out of scope for this particular PR.

**Comment by ibusko** [comment]
I agree that the images argument should be optional and default to None, but only if that is the most frequent way users are going to use MosViz. From my previous interaction with this app, I thought the intended default way is to be used with images (as when reading from a mos table), and in that case the no-images mode would be optional. 

Maybe we need PO advice here?

**Comment by duytnguyendtn** [comment]
I'm personally in the opinion that the software should try its best given what the user gives it. In that case, I'd be in opinion of not requiring `images=None`

**Comment by ibusko** [comment]
I modified `load_data` to have 'images=None' by default.

**Review by rosteen — APPROVED** [review]
Looks good to me

File: notebooks/MosvizExample.ipynb [hunk 1]
```diff
@@ -59,24 +59,71 @@
...
+    "    images.append(str(level3_path / 'mosviz_cutouts' / '227.fits'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mosviz.load_data(spectra_1d, spectra_2d, images=images)\n",
```
Comment (rosteen) [inline notebooks/MosvizExample.ipynb:72]: Just a note that you can still use the old syntax here, you don't necessarily have to name the images argument. Maybe we actually want to keep it as-is (I say, having already approved the PR)?

Comment (ibusko) [reply notebooks/MosvizExample.ipynb:72]: On the other hand, writing it explicitly ensures the user gets the point that it's an optional argument.

Comment (rosteen) [reply notebooks/MosvizExample.ipynb:72]: Good point! Merging...

**Pull request merged**